### DISCUSSION TITLE
Pivot Grid - Columns can not be removed (rearrenged)

### DISCUSSION THREAD

**Discussion by DataEngineerVienna** [TabularEditor/TabularEditor3#1050] [Ideas]

**Comment by otykier** [comment]
You can rearrange or remove columns by dragging them from the header areas of the pivot table like so:

![pivotgrid](https://user-images.githubusercontent.com/8976200/122756214-69fc4580-d296-11eb-8d66-c64480bf7a55.gif)

**Comment by DataEngineerVienna** [comment]
ok, thanks! Worked
(I tried to drag the column header...)
A bit confusing as I only see some white boxes without titles...!?

![grafik](https://user-images.githubusercontent.com/61802471/122756466-be9fc080-d296-11eb-9d30-093fbe5daeed.png)
